### PR TITLE
Note on adding SSH key to the ssh-agent

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

### CODE REVIEW

File: desktop/dev-environments.md [hunk 1]
```diff
@@ -35,10 +35,14 @@ Click **Install** to download and install any missing tools.
 
 ## Start a single container Dev Environment
 
-The simplest way to get started with Dev Environments is to create a new environment by cloning the Git repository of the project you are working on. For example, let us create a new Dev Environment using a simple `singe-dev-env` project from the [Docker Samples](https://github.com/dockersamples/single-dev-env){:target="_blank" rel="noopener" class="_"} GitHub repository.
+The simplest way to get started with Dev Environments is to create a new environment by cloning the Git repository of the project you are working on. For example, let us create a new Dev Environment using a simple `single-dev-env` project from the [Docker Samples](https://github.com/dockersamples/single-dev-env){:target="_blank" rel="noopener" class="_"} GitHub repository.
+
+> **Note**
+>
+> When cloning a Git repository using SSH, ensure you've added your SSH key to the ssh-agent. To do this, open a terminal and run `ssh-add $HOME/.ssh/id_rsa`.
```
Comment (rumpl) [inline desktop/dev-environments.md:42]: The key isn't always `id_rsa`, maybe we should put `ssh-add <PATH>` or some other way of saying `ssh-add <path to yout public ssh key>` ?

Comment (usha-mandya) [reply desktop/dev-environments.md]: Updated

**Review by rumpl — APPROVED** [review]
LGTM thanks!

**Review by jerae-duffin — COMMENTED** [review]
LGTM

**Review by jerae-duffin — APPROVED** [review]

**Pull request merged**